test/serializer/basic/PreventExtensions.js

```
@@ -0,0 +1,6 @@
+let obj = {x:1};
+Object.preventExtensions(obj);
+inspect = function() {
+  obj.y = 1;
```

It's silently ignored in non-strict mode. In strict-mode it throws.